File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
       int n=nums.size();
       for(int i=0;i<n;i++){
        int x=nums[i];
        int y=target-x;
        for(int j=i+1;j<n;j++){
            if(nums[j]==y){
                return {i,j};
            }
        }
       }
       return {};
    }
};

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let n=nums.length;
    let map=new Map();
    for(let i=0;i<n;i++){
       let need=target-nums[i];
       if(map.has(need)) return [i,map.get(need)];
       map.set(nums[i],i);
    }
    return [];
};